groupnews-web/src/test/java/com/epam/internship/UserControllerTest.java

```diff
@@ -0,0 +1,92 @@
+package com.epam.internship;
+
+import java.util.Arrays;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+
+import com.epam.internship.controller.UserController;
+import com.epam.internship.dto.User;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(controllers = {UserController.class})
+public class UserControllerTest {
+
+	@Autowired
+	private MockMvc mockMvc;
+	
+	@MockBean
+	private UserService systemUnderTest;
```

Comment:
UserService is not the subject of your test, so please do not call it 'systemUnderTest'. It is misleading.